client/src/main/java/jp/co/soramitsu/iroha/java/QueryAPI.java

```diff
@@ -181,13 +182,31 @@ public BlockResponse getBlock(Long height) {
 
     return res.getBlockResponse();
   }
+//  extended TxPaginationMeta
+  public TransactionsPageResponse getAccountTransactions(String accountId,
+                                                         Integer pageSize,
+                                                         String firstHashHex,
+                                                         QueryBuilder.Ordering ordering,
+                                                         Timestamp firstTxTime,
+                                                         Timestamp lastTxTime,
+                                                         Integer firstTxHeight,
+                                                         Integer lastTxHeight) {
+    val q = Query.builder(this.accountId, counter.getAndIncrement(), signatureBuilder)
+            .getAccountTransactions(accountId, pageSize, firstHashHex, ordering, firstTxTime, lastTxTime, firstTxHeight, lastTxHeight)
+            .buildSigned(keyPair);
 
+    val res = api.query(q);
+
+    checkErrorResponse(res);
+
+    return res.getTransactionsPageResponse();
```

Comment:
I think You can use another function instead of those lines
```
return getAccountTransactions(accountId, pageSize, firstHashHex, ordering, null, null, null, null)
```